### PR TITLE
PLATFORM-1310: Move user history related tables to specials

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/PLATFORM-1310

`user_login_history_summary` is used in joins with `events_local_users` - they both need to be on specials DB.

To keep all related tables in the same place let's move these tables from stats to specials:
- user_history (2.4 mm rows)
- user_login_history (500 mm rows)
- user_login_history_summary (6.7 mm rows)

@michalroszka / @wladekb / @drozdo

### CODE REVIEW

File: extensions/wikia/UserChangesHistory/UserChangesHistory.class.php [hunk 1]
```diff
@@ -70,31 +70,35 @@ static public function LoginHistoryHook( $from, $user, $type = false ) {
 							WScribeClient::singleton('trigger')->send($data);
 						}
 						catch( TException $e ) {
-							Wikia::log( __METHOD__, 'scribeClient exception', $e->getMessage() );
+							Wikia\Logger\WikiaLogger::instance()->error( __METHOD__, [
```
Comment (wladekb) [inline extensions/wikia/UserChangesHistory/UserChangesHistory.class.php]: I'd suggest not removing the message "scribeClient exception"

**Comment by wladekb** [comment]
it makes more sense to keep `user_login_history` and `user_login_history_summary` in the same db, can we move the `user_login_history` to `specials` as well?

**Comment by macbre** [comment]
`user_login_history_summary` has 6 mm and `user_login_history` 500 mm rows. I'd like to move as less data as possible. I'll consult that with @drozdo

**Comment by macbre** [comment]
@wladekb: Drozdo is moving all three tables (listed in PR description) from stats to specials DB. PR updated accordingly.

**Comment by wladekb** [comment]
Cool! Thanks!

**Comment by michalroszka** [comment]
:+1:

**Comment by macbre** [comment]
Thanks for the CR, I'll merge it into dev and push it live when we're done with tables migration (still 2-3 days left).

**Pull request merged**